test/fixtures/images/runner/forever.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"time"
+)
+
+func main() {
+	for {
+		time.Sleep(1 * time.Minute) // Sleep for 1 hour in each iteration
+	}
+}
```

Comment:
why not just use an alpine container who's entrypoint is `sleep 1000d`?

Reply:
I'd like to test pulling images from our Registry. And the most of all, it is really easy to hit the docker.io daily limit if/when the tests are executed quite often, which is the case during development phase.

Reply:
Probably, we could pull the alpine image or even something smaller from docker.io once push it to the registry running withing the compose test env.